File: java/src/main/java/com/dddeurope/niuz/events/ArticleSubmitted.java
package com.dddeurope.niuz.events;

public class ArticleSubmitted {
    private final String authorId;
    private final String headline;

    public ArticleSubmitted(String authorId, String headline) {
        this.authorId = authorId;
        this.headline = headline;
    }

    public String getAuthorId() {
        return authorId;
    }

    public String getHeadline() {
        return headline;
    }
}

File: dotnet/niuz.application/events/ArticleSubmitted.cs
namespace niuz.application.events
{
    public class ArticleSubmitted
    {
        public ArticleSubmitted(string authorId, string headline)
        {
            AuthorId = authorId;
            Headline = headline;
        }

        public string AuthorId { get; }
        public string Headline { get; }
    }
}